akka-docs/rst/java/cluster-sharding.rst

```diff
@@ -200,6 +200,8 @@ on all nodes::
 You must explicitly add the ``akka-distributed-data-experimental`` dependency to your build if
 you use this mode. It is possible to remove ``akka-persistence`` dependency from a project if it
 is not used in user code and ``remember-entities`` is ``off``.
+Using it together with ``Remember Entities`` shards will be recreated after rebalancing, however will
```

Comment:
There is a sentence that remember Entities keeps data in a persistence - and that's fine. However when you switch stateStoreMode to ddata (in our case for test purposes, globally) rememberEntities stopped working at start, but after a wake-up it was working fine (so you need to send message to one actor in a region, then all of them were started)

That was my intention to add this remark